angelsexploration/prototypes/entities/biter-builder.lua

```diff
@@ -830,10 +830,9 @@ function angelsmods.functions.make_alien(def_data)
   --log(serpent.block(def_data))
   if def_data ~= nil then
     local c_name = def_data.appearance.name .. "-" .. def_data.appearance.type .. "-corpse"
+    local c_type = "big-biter-corpse"
     if def_data.appearance.type == "psyker" or def_data.appearance.type == "psyker" then
```

Comment:
This seems to have 2x `def_data.appearance.type == "psyker"`, shouldn't the 2nd one be the other type of biter in angels?